command/update/command.go

```diff
@@ -196,56 +195,10 @@ func (c *Command) execute() error {
 			return microerror.Mask(err)
 		}
 
-		c.logger.Log("debug", fmt.Sprintf("added IP to endpoint of service '%s'", f.Kubernetes.Cluster.Service))
-	}
-
-	// Listen to OS signals issued by the Kubernetes scheduler.
-	signalCh := make(chan os.Signal, 2)
-	signal.Notify(signalCh, syscall.SIGTERM, syscall.SIGKILL)
-
-	c.logger.Log("debug", "waiting for termination signal")
-
-	s := <-signalCh
-
-	c.logger.Log("debug", fmt.Sprintf("received termination signal: %#v (%s)", s, s))
-
-	errCh := make(chan error)
-
-	// Use the updater to actually delete the endpoints identified by the provided
-	// flags.
-	go func() {
-		defer close(errCh)
-
-		action := func() error {
-			err := newUpdater.Delete(f.Kubernetes.Cluster.Namespace, f.Kubernetes.Cluster.Service, podInfos)
-			if err != nil {
-				return microerror.Mask(err)
-			}
-			return nil
-		}
-
-		err := backoff.Retry(action, backoff.NewExponentialBackOff())
-		if err != nil {
-			errCh <- microerror.Mask(err)
-			return
-		}
-
-	}()
-
-	exitCode := 0
-
-	select {
-	case s := <-signalCh:
-		c.logger.Log("debug", fmt.Sprintf("received termination signal: %#v (%s) again, exiting immediately", s, s))
-	case err := <-errCh:
-		if err != nil {
-			c.logger.Log("error", fmt.Sprintf("%#v", microerror.Mask(err)))
-			exitCode = 1
-		}
+		c.logger.Log("debug", fmt.Sprintf("added annotations to the KVM pod '%s'", f.Kubernetes.Pod.Name))
 	}
 
-	c.logger.Log("debug", fmt.Sprintf("shutting down with exit code %d", exitCode))
-	os.Exit(exitCode)
+	c.logger.Log("debug", fmt.Sprint("shutting down "))
```

Comment:
I guess we have to keep the process running since we are not doing in the KVM operator where we invoke the container. We can just use `select{}` to do this. This logging is then meaningless.

Reply:
fixed